Makefile

```diff
@@ -138,6 +138,21 @@ test-eval: ## Run agent evaluation tests
 	sed -i "s/uniq-cluster-name/$${UNIQUE_ID}/g" $$TEMP_DIR/eval_data.yaml
 	cd test/evals && python eval.py --eval_data_yaml $$TEMP_DIR/eval_data.yaml
 
+.ONESHELL:
+qe-test-eval: ## Run agent QE evaluation tests
```

Comment:
Can we share code with the `test-eval` target? The only difference is the YAML file